static_src/stores/base_store.js

```diff
@@ -78,6 +78,17 @@ export default class BaseStore extends EventEmitter {
     return !Immutable.is(this._data, c);
   }
 
+  deleteProp(guid, prop, cb = defaultChangedCallback.bind(this)) {
+    const index = this._data.findIndex((d) =>
+      d.get('guid') === guid
+    );
+
+    if (index === -1) return cb(false);
+
+    this._data = this._data.deleteIn([index, prop]);
```

Comment:
This was the key thing missing before. I got so caught up with the `_data` that I didnt consider the issue was in how the `_data` was being rendered. Thanks for synthesizing this.